src/ClimaLSM.jl

```diff
@@ -45,25 +39,35 @@ function RootSoilModel{FT}(;
     vegetation_model_type::Type{VM},
     vegetation_args::NamedTuple = (;),
 ) where {FT, SM <: AbstractSoilModel{FT}, VM <: AbstractVegetationModel{FT}}
-    #configuration may depend on the versions of the soil and root models too
-    configuration = Configurations.RootSoilConfiguration{FT}()
-    soil = soil_model_type(; configuration = configuration, soil_args...)
+    sources = (RootExtraction{FT}(),)
+    boundary_fluxes = FluxBC{FT}(FT(0.0), FT(0.0))
+    transpiration = PrescribedTranspiration{FT}((t::FT) -> FT(0.0))
+    root_extraction = PrognosticSoilPressure{FT}()
+    soil = soil_model_type(;
+        boundary_conditions = boundary_fluxes,
+        sources = sources,
+        soil_args...,
+    )
     vegetation = vegetation_model_type(;
-        configuration = configuration,
+        root_extraction = root_extraction,
+        transpiration = transpiration,
         vegetation_args...,
     )
-    args = (configuration, soil, vegetation)
+    args = (soil, vegetation)
     return RootSoilModel{FT, typeof.(args)...}(args...)
 end
 
 function initialize_interactions(
-    land::RootSoilModel{FT, RootSoilConfiguration{FT}},
-) where {FT}
+    land::RootSoilModel{
```

Comment:
@jb-mackay this is what I was mentioning on slack....we need to put in placeholders for all the parametric types of Root and Soil models...

not sure why